### PR TITLE
Add flux injection

### PR DESCRIPTION
**Pull request by RemiLehe**

### CODE REVIEW

File: PICMI_Python/particles.py [hunk 1]
```diff
@@ -282,6 +282,57 @@ def __init__(self, density,
...
+                 **kw):
+        self.flux = flux
+        self.flux_normal_axis = flux_normal_axis
+        self.surface_flux_position = surface_flux_position
+        self.flux_direction = flux_direction
+        self.lower_bound = lower_bound
+        self.upper_bound = upper_bound
+        self.rms_velocity = rms_velocity
+        self.directed_velocity = directed_velocity
+        self.fill_in = fill_in
```
Comment (dpgrote) [inline PICMI_Python/particles.py:332]: This line is presumably left over from a copy and paste. Otherwise, this looks good.

Comment (RemiLehe) [inline PICMI_Python/particles.py:332]:
```suggestion
```

**Comment by RemiLehe** [comment]
Thanks! I made the corresponding change. Do you want to merge this?

**Review by dpgrote — APPROVED** [review]
Thanks! Looks good!

**Pull request merged**